File: Client/Domain/Pair.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace app.Domain
{ [Serializable]
   public class Pair
    {
        public Racer Racer { get; set; }
        public Race Race { get; set; }

        public Pair(Racer racer,Race race)
        {
            this.Racer = racer;
            this.Race = race;
        }
    }
}

File: Server/AppDomain/src/main/java/app/Domain/Pair.java
package app.Domain;

import java.io.Serializable;

public class Pair implements Serializable {
    private Racer racer;
    private Race race;

    public Pair(Racer racer, Race race) {
        this.racer = racer;
        this.race = race;
    }

    public Racer getRacer() {
        return racer;
    }

    public void setRacer(Racer racer) {
        this.racer = racer;
    }

    public Race getRace() {
        return race;
    }

    public void setRace(Race race) {
        this.race = race;
    }
}
